Spell_Chech_Example.c

```diff
@@ -38,43 +39,67 @@ _spell_check_word(const char* str)
 
 
 /************ EFL Spell Check Logic ******************/
+
+static Efl2_Text_Attribute_Handle*
+mark_misspelled(Eo *start,Eo *end)
+{
+   efl2_text_style_underline_color_set(attribute_factory, 255, 0, 0, 255);
+   Efl2_Text_Attribute_Handle handle = efl2_text_attribute_factory_insert(attribute_factory, start, end);
+   Efl2_Text_Attribute_Handle *handle_ref = malloc(sizeof(Efl2_Text_Attribute_Handle));
```

Comment:
Definitely no. You can't alloc it on your own even if you wanted, but also you don't want to.